src/software/ai/hl/stp/play/corner_kick_play.cpp

```diff
@@ -66,6 +66,51 @@ void CornerKickPlay::getNextTactics(TacticCoroutine::push_type &yield, const Wor
     auto goalie_tactic = std::make_shared<GoalieTactic>(
         world.ball(), world.field(), world.friendlyTeam(), world.enemyTeam());
 
+    // Setup two bait robots on the opposite side of the field to where the corner kick
+    // is taking place to pull enemies away from the goal
+    Point opposite_corner_to_kick = kick_from_pos_corner ? world.field().enemyCornerNeg()
```

Comment:
Random idea: If you want to pull robots to the side of the field the kick is being taken from, ie. the side where the ball is, you could just `copysign` the ball rather than have this variable.

Reply:
I've removed the variable, but I don't think that `copysign` would be an improvement